projects/libhtp-rs/cover.sh

```diff
@@ -0,0 +1,10 @@
+mkdir /cover
```

Comment:
We could do it this way :

    $ sydr-fuzz -c sydr-fuzz.toml cov-export -- -format=lcov > htp.lcov

Script is not needed

Reply:
i had some problems with llvm-profdata merge.
I hope it was just a mistake on my part.